apps/gateway/.env.example

```
@@ -1,19 +1,68 @@
+GATEWAY_ENCRYPTION_KEY=
 
-# ── WhatsApp Baileys Adapter (F3a-21) ────────────────────────────────────────
-WA_SESSIONS_DIR=./data/wa-sessions   # Directorio para auth state de Baileys
-                                      # Un subdirectorio por channelConfigId
-                                      # NUNCA commitear este directorio
-WA_MAX_RECONNECT_ATTEMPTS=5          # Intentos de reconexión antes de closed
+# =============================================================================
+# Rate limiting
+# =============================================================================
+WEBHOOK_RATE_LIMIT=300
+API_RATE_LIMIT=120
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_RATE_LIMIT key should go before the WEBHOOK_RATE_LIMIT key

(UnorderedKey)